app/src/main/java/org/openedx/app/MainViewModel.kt

```diff
@@ -69,4 +75,18 @@ class MainViewModel(
             }
         )
     }
+
+    private fun logSettingPermissionStatusEvent() {
```

Comment:
IMO, `logNotificationPermissionStatusEvent` is more appropriate.

Reply:
renamed.